etc/init.d/contextBroker.centos

```diff
@@ -91,7 +91,10 @@ broker_start()
 {
     printf "%-50s" "Starting ${NAME}..."
     if [[ -x ${EXECUTABLE} ]]; then
-        su $BROKER_USER -c "${EXECUTABLE} ${BROKER_OPTS} &> /dev/null "
+        # preserving existing error log (if it already exists)
+        LOG_FILE_BASE=${BROKER_LOG_DIR}.${EXECUTABLE}
+        su $BROKER_USER -c "mv ${LOG_FILE_BASE}.err ${LOG_FILE_BASE}.err.old 2> /dev/null"
+        su $BROKER_USER -c "${EXECUTABLE} ${BROKER_OPTS} > /dev/null 2> ${LOG_FILE_BASE}.err"
         sleep 2 # wait some time to leave contextbroker start
```

Comment:
Now that we're here ...

   wait some time to leave contextbroker ...   (Spanish: dejar el broker arrancar)

In English it would be "to **let** the broker start"